lib/hooks/index.js

```diff
@@ -40,3 +40,4 @@ export * from './useTranslatedCategories';
 export * from './useUser';
 export * from './useUsersBatch';
 export * from './useLocalPagination';
+export * from './useVersionHistoryValueResolvers';
```

Comment:
please keep it in the `VersionHistory` level hooks https://github.com/folio-org/stripes-acq-components/tree/master/lib/VersionHistory/hooks

Reply:
Done